src/test/java/uk/gov/defra/datareturns/test/licence/LicenceLookupIT.java

```diff
@@ -54,11 +54,16 @@ public void testLicenceReadOnly() {
     }
 
     @Test
-    public void testLFullicenceLookupB7A111() {
+    public void testLFullLicenceLookupB7A111() {
         getEntity("/licence/full/00081019-1WS3JP4-B7A718")
                 .statusCode(HttpStatus.OK.value());
     }
 
+    @Test
+    public void testFullLicenceLookupInvalid() {
```

Comment:
I think it's probably worth having a few invalid test cases (similarly, above, have a few valid options too). Improves the use-case examples and possibly covers some edge cases too